apps/dialtone-documentation/docs/components/scrollbar.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Scrollbar
+description: A directive that adds customized overlay scrollbars to any scrollable section.
+status: beta
+thumb: true
+image: assets/images/components/scroller.png
+---
+
+## Scrollbar directive
+
+Allows to add overlay scrollbars that will look the same for every browser. The directive sets up the scrollbars from the library [OverlayScrollbars](https://kingsora.github.io/OverlayScrollbars/).
+
+<code-well-header>
+  <div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300" v-dt-scrollbar>
+    <div v-for="item in items" class="item">
+      {{ item}}
+    </div>
+  </div>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300 custom-scrollbars" data-overlayscrollbars="host" data-overlayscrollbars-initialize="true">
+  <div class="os-size-observer">
+    <div class="os-size-observer-listener"></div>
+  </div>
+  <div data-overlayscrollbars-viewport="scrollbarHidden overflowXHidden overflowYScroll" tabindex="-1">
+    <div class="item">user 1</div>
+    ...
+  </div>
+  <div class="os-scrollbar os-scrollbar-horizontal os-theme-dark os-scrollbar-auto-hide os-scrollbar-handle-interactive os-scrollbar-cornerless os-scrollbar-unusable">
+    <div class="os-scrollbar-track">
+      <div class="os-scrollbar-handle"></div>
+    </div>
+  </div>
+  <div class="os-scrollbar os-scrollbar-vertical os-theme-dark os-scrollbar-auto-hide os-scrollbar-handle-interactive os-scrollbar-visible os-scrollbar-cornerless">
+    <div class="os-scrollbar-track">
+      <div class="os-scrollbar-handle"></div>
+    </div>
+  </div>
+</div>
+'
+vueCode='
+<div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300" v-dt-scrollbar>
+  <div v-for="item in items" class="item">
+    {{ item}}
+  </div>
+</div>
+'
+/>
+
+## Usage
+
+Import the directive from dialtone-vue
+
+```javascript
+import { DtScrollbarDirective } from "@dialpad/dialtone-vue";
+```
+
+Install the directive into vue instance
+
+```javascript
+Vue.use(DtScrollbarDirective);
+```
+
+To add customized overlay scrollbars to a scrollable section, simply include the `v-dt-scrollbar` directive on the desired element.
+Add a `max-height` to the element if you want to set up a vertical scrollbar, or a `max-width` if you want to set up a horizontal scrollbar.
```

Comment:
This sounds like it's required. Is it? 

I imagine most scrollable regions won't have a `width` or `max-width` declared and are just block-level elements filling its containing width. Probably same for height, especially if they're flex items that grow.

Reply:
You are right. Changed it to "There is no need to explicitly add an `overflow` property. If the section overflows the available vertical space, a vertical scrollbar will be present. Similarly, if it exceeds the horizontal space, a horizontal scrollbar will appear."